examples/gnrc_networking/Makefile

```diff
@@ -34,9 +34,19 @@ USEMODULE += netstats_l2
 USEMODULE += netstats_ipv6
 USEMODULE += netstats_rpl
 
-# Set a custom 802.15.4 channel if needed
-DEFAULT_CHANNEL ?= 26
-CFLAGS += -DDEFAULT_CHANNEL=$(DEFAULT_CHANNEL)
+# Set a custom channel if needed
+ifeq (,$(filter cc110x,$(USEMODULE)))     # radio is cc110x sub-GHz
```

Comment:
Ahh, sorry yes! You need to move it below l. 63 (after the include of `$(RIOTBASE)/Makefile.include`)